src/gui_common/art_gallery/CrossFadableGalleryViewport.cs

```diff
@@ -0,0 +1,62 @@
+﻿using Godot;
+
+/// <summary>
+///   Handles fade effects in the gallery's model viewer. For generic fades, see
+///   <see cref="CrossFadableTextureRect"/> or <see cref="ScreenFade"/>.
+/// </summary>
+public class CrossFadableGalleryViewport : ViewportContainer
+{
+    private bool fadeRequested;
+
+#pragma warning disable CA2213
+    private Tween tween = null!;
+#pragma warning restore CA2213
+
+    [Signal]
+    public delegate void Faded();
+
+    /// <summary>
+    ///   Triggers the gallery model viewer's crossfade effect.
+    /// </summary>
+    public bool FadeRequested
```

Comment:
Is there any reason to use the FadeRequested property instead of letting SlideScreen call BeginFade() directly?

Reply:
I had a look through the changes here and this is the only thing I also noticed that could be improved.

Reply:
I tried this, but I don't really see a way to just call the method without running into errors about conflicting static and non-static contexts. ChangeSlide() is static, while BeginFade() needs to be nonstatic. Most direct I can do is probably a signal.

Reply:
I might have worded my initial review badly. You don't have to call BeginFade() in a static way. You can just call modelViewContainer!.BeginFade()